src/cli/tests/test_app.py

```
@@ -0,0 +1,724 @@
+    def test_create_command(self):
+        parser = self._build_parser()
+        args = parser.parse_args(['app', 'create', 'my-app', '-d', 'Description'])
+        self.assertEqual(args.command, 'create')
+        self.assertEqual(args.name, 'my-app')
+        self.assertEqual(args.description, 'Description')
+        self.assertIsNone(args.file)
+
+    def test_create_command_with_file(self):
+        parser = self._build_parser()
+        args = parser.parse_args(
+            ['app', 'create', 'my-app', '-d', 'desc', '-f', '/tmp/spec.yaml'])
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/spec.yaml"

(S108)